.evergreen/provision-atlas.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+PYTHON_BINARY=/opt/python/3.10/bin/python3
+
+# Should be called from src
+EVERGREEN_PATH=$(pwd)/.evergreen
+TARGET_DIR=$(pwd)/$DIR
+PING_ATLAS=$EVERGREEN_PATH/ping_atlas.py
+SCAFFOLD_SCRIPT=$EVERGREEN_PATH/scaffold_atlas.py
+DEPLOYMENT_NAME=$DIR
+
+# Download the mongodb tar and extract the binary into the atlas directory
+# TODO: make this work for macos and linux
+set -ex
+curl https://fastdl.mongodb.org/mongocli/mongodb-atlas-cli_1.14.0_linux_x86_64.tar.gz -o atlas.tgz
+tar zxf atlas.tgz
+mv mongodb-atlas-cli_1.14.0* atlas
+
+# Install Podman based on OS
+set -Eeou pipefail
+if [ -n "$(which yum 2>/dev/null)" ]; then
+    sudo yum install -y podman
```

Comment:
We're discouraged from installing global binaries on the hosts.

Reply:
Removed and upgraded distro to `rhel87-small`